### ISSUE TITLE
Open ui to player

### ISSUE THREAD

**Issue by GoldenMyst** [Camotoy/floodgate-skript#11]
Open a specific ui to the player like “How to play?”, “marketplace”, “settings”, etc.

**Comment by zimzaza4** [comment]
https://github.com/Camotoy/floodgate-skript/issues/2

**Comment by Camotoy** [comment]
Nah, this is slightly different which is why I kept it open. As opposed to forms, this one requests menus to be opened, which may not be super easy, unfortunately.

**Comment by zimzaza4** [comment]
Just like "server settings"?

**Comment by Camotoy** [comment]
That is the interpretation I got from this.
